rsconnect/api.py

```diff
@@ -520,6 +520,46 @@ class ServerDetails(TypedDict):
     python: ServerDetailsPython
 
 
+class ExecutorKwargs(TypedDict):
```

Comment:
I'd be very happy if no functions in this module accepted `**kwargs`.

Reply:
Agreed, that is a good goal.